schedule/containers/extra_tests_textmode_containers.yaml

```diff
@@ -1,27 +1,37 @@
 name:           extra_tests_textmode_containers
 description:    >
-    Maintainer: jalausuch@suse.com, qa-c@suse.de.
-    Extra tests about software in containers module
+  Maintainer: jalausuch@suse.com, qa-c@suse.de.
```

Comment:
feel free to remove my name here and keep the team ML

Reply:
7f7d1aa26